demo/readFormFields.js

```diff
@@ -26,6 +26,31 @@ function GetSelectorForElement(el) {
   return queryParts.join(' > ');
 }
 
+function ReadCookies() {
+  var cookies = document.cookie;
+  var coolist = cookies.split(' ');
+  if (document.getElementById('ins_cookies').innerHTML !== '') {
+    document.getElementById('ins_cookies').innerHTML = '';
+  } else {
+    for (const cookie of coolist) {
```

Comment:
This is JS, you should add this code to the TS version of this.
Also, this file is readFormField, not readCookie. 
Seems better to add another file.